scripts/crosstool-NG.sh.in

```diff
@@ -60,6 +60,9 @@ CT_TestAndAbort "Don't set CPLUS_INCLUDE_PATH. It screws up the build." -n "${CP
 CT_TestAndAbort "Don't set OBJC_INCLUDE_PATH. It screws up the build." -n "${OBJC_INCLUDE_PATH}"
 CT_TestAndAbort "Don't set CFLAGS. It screws up the build." -n "${CFLAGS}"
 CT_TestAndAbort "Don't set CXXFLAGS. It screws up the build." -n "${CXXFLAGS}"
+CT_TestAndAbort "Don't set CC.  Support is planned, but currently it screws up the build. Ensure the GCC you
```

Comment:
Please just use the same message as the rest of the lines.
- I don't have a definitive date when this support is going to happen, so this will raise more questions than answer.
- It is possible to use a differently named GCC even now, as evidenced by the FreeBSD port (#627), even though it is not widely tested yet